src/vma/dev/ring_bond.cpp

```diff
@@ -394,6 +394,7 @@ bool ring_bond::reclaim_recv_buffers(descq_t *rx_reuse)
 {
 	devide_buffers_helper(rx_reuse, m_buffer_per_ring);
 	for (uint32_t i = 0; i < m_n_num_resources; i++) {
+		m_buffer_per_ring[i].clear_without_cleanup();
```

Comment:
clear_without_cleanup() will make size() to return 0. Do we expect the next if statement to succeed?
```
if (m_buffer_per_ring[i].size() > 0) {
```